tests/components/roku/test_media_player.py

```diff
@@ -213,118 +239,126 @@ async def test_tv_attributes(hass: HomeAssistantType, requests_mock: Mocker) ->
     state = hass.states.get(TV_ENTITY_ID)
     assert state.state == STATE_PLAYING
 
-    assert state.attributes.get(ATTR_MEDIA_CONTENT_TYPE) == MEDIA_TYPE_CHANNEL
+    assert state.attributes.get(ATTR_APP_ID) == "tvinput.dtv"
+    assert state.attributes.get(ATTR_APP_NAME) == "Antenna TV"
     assert state.attributes.get(ATTR_INPUT_SOURCE) == "Antenna TV"
+    assert state.attributes.get(ATTR_MEDIA_CONTENT_TYPE) == MEDIA_TYPE_CHANNEL
+    assert state.attributes.get(ATTR_MEDIA_CHANNEL) == "getTV (14.3)"
+    assert state.attributes.get(ATTR_MEDIA_TITLE) == "Airwolf"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)